<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSFWindow</class>
 <widget class="QMainWindow" name="CSFWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSFWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>971</width>
      <height>671</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_input">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QPushButton" name="Segmentation">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tissu Segmentation </string>
      </property>
     </widget>
     <widget class="QPushButton" name="CSF_Probability_Map">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CSF Probability Map</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_T1img">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>20</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Segmentation">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>70</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CSF_Probability_Map">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>120</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="left_hemisphere">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Left Hemisphere</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LH_MID_Surface">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LH MID Surface</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_LH_MID_Surface">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>200</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="LH_GM_Surface">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LH GM Surface</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_LH_GM_Surface">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>250</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="right_hemisphere">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Right Hemisphere</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RH_MID_Surface">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RH MID Surface</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_RH_MID_Surface">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>330</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RH_GM_Surface">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RH GM Surface</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_RH_GM_Surface">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>380</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="outputdirectory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Directory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="output_directory">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>470</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Directory</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Output_Directory">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>470</y>
        <width>531</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="T1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>T1 image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_executables">
     <attribute name="title">
      <string>Executables</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_exection">
     <attribute name="title">
      <string>Execution</string>
     </attribute>
     <widget class="QPushButton" name="Execute">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="output">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>351</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="error">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>150</y>
        <width>361</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Output messages</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Errors</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="local">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Option 1: Local</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="longleaf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Option 2: Longleaf</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoad_Config_File"/>
    <addaction name="actionSave_Config_File"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad_Config_File">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Config File</string>
   </property>
  </action>
  <action name="actionSave_Config_File">
   <property name="text">
    <string>Save Config File</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
